src/main/java/LadderController.java

```diff
@@ -0,0 +1,78 @@
+import domain.Height;
+import domain.PersonCount;
+import domain.ladder.Ladder;
+import domain.ladder.RandomLinkGenerator;
+import domain.prize.Prizes;
+import domain.user.Users;
+import java.util.List;
+import java.util.stream.Collectors;
+import view.InputView;
+import view.LineRender;
+import view.OutputView;
+
+public class LadderController {
+
+    private static final int MAX_TRY_COUNT = 5;
+
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+
+    public void run() {
+        int tryCount = 0;
+        final Users users = generateUsers();
+        final Ladder ladder = generateLadder(users);
+        final Prizes prizes = generatePrizes(users);
+        printGenerateLadderResult(ladder, users, prizes);
+        ladder.playLadderGame(users);
+        while (tryCount++ < MAX_TRY_COUNT) {
```

Comment:
```suggestion
        int tryCount = 0;
        final Users users = generateUsers();
        final Ladder ladder = generateLadder(users);
        final Prizes prizes = generatePrizes(users);
        
        printGenerateLadderResult(ladder, users, prizes);
        
        ladder.playLadderGame(users);
        
        while (tryCount++ < MAX_TRY_COUNT) {
```

메서드 내부의 적절한 개행은 코드 가독성을 높여줍니다 !